### PR TITLE
manix: 0.6.2 -> 0.6.3

### PR DESCRIPTION
**Pull request by mlvzk**
###### Motivation for this change

Update manix to latest version 0.6.3.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by legendofmiracles** [comment]
Result of `nixpkgs-review pr 119912` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>manix</li>
  </ul>
</details>

File: pkgs/tools/nix/manix/default.nix [hunk 1]
```diff
@@ -2,18 +2,18 @@
...
 
   src = fetchFromGitHub {
     owner = "mlvzk";
     repo  = pname;
     rev = "v${version}";
-    sha256 = "0fv3sgzwjsgq2h1177r8r1cl5zrfja4ll801sd0bzj3nzmkyww7p";
+    sha256 = "1b7xi8c2drbwzfz70czddc4j33s7g1alirv12dwl91hbqxifx8qs";
   };
 
   buildInputs = lib.optional stdenv.isDarwin [ darwin.Security ];
```
Comment (SuperSandro2000) [inline pkgs/tools/nix/manix/default.nix:14]:
```suggestion
  buildInputs = lib.optional stdenv.isDarwin [ Security ];
```

Comment (mlvzk) [reply pkgs/tools/nix/manix/default.nix]: Done https://github.com/NixOS/nixpkgs/pull/119912/commits/e1db02b6a769ebab390c09d46948f7ea2c9875a4

File: pkgs/tools/nix/manix/default.nix [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
   src = fetchFromGitHub {
     owner = "mlvzk";
     repo  = pname;
     rev = "v${version}";
-    sha256 = "0fv3sgzwjsgq2h1177r8r1cl5zrfja4ll801sd0bzj3nzmkyww7p";
+    sha256 = "1b7xi8c2drbwzfz70czddc4j33s7g1alirv12dwl91hbqxifx8qs";
   };
 
-  buildInputs = lib.optional stdenv.isDarwin [ darwin.Security ];
+  buildInputs = lib.optional stdenv.isDarwin [ Security ];
```
Comment (SuperSandro2000) [inline pkgs/tools/nix/manix/default.nix:14]:
```suggestion
  buildInputs = lib.optionals stdenv.isDarwin [ Security ];
```

**Pull request merged**